### DISCUSSION TITLE
Feedback!

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [Cyfrin/moccasin-full-course-cu#4] [General]
Please leave feedback here!

- What parts of the course were too hard?
- Too confusing?
- Do you wish there was a better format?
- What parts did you LOVE?

All feedback is welcome!

**Comment by s3bc40** [comment]
Hey! Thanks for your work and this amazing course. 

Python was my first programming language, and it gave me the opportunity to go above and beyond.
Getting back to it for Web3 is a great pleasure!

Another subject, I happen to see some mistakes on the writing part of the course (even though they are minor issues) like:
```diff
- name: public(string[100])
+ name: public(String[100])
```

I have always wondered how I could give a direct feedback. Maybe creating a discussion with all my findings would be better for your team?

I am getting back to the course and thanks again to all the Cyfrin team!

**Reply by urbanvybz3-ai** [reply]
do u have online classes?

**Reply by EngrPips** [reply]
Hmmmm 🤔

**Comment by 0xk3ysk1lls** [comment]
Hi,
it seems like lessons 13-15 (Section 2: Web3py) link to the same video: [https://updraft.cyfrin.io/courses/intermediate-python-vyper-smart-contract-development/web3py/tx](https://updraft.cyfrin.io/courses/intermediate-python-vyper-smart-contract-development/web3py/tx). There are no links with ending .../tx2 nor ...tx/3 like there are with Anvil lessons. Hence the bug (probably).

And btw I just want to thank you for your amazing work and great tutorial!

**Reply by iburr** [reply]
Had the same issue try clearing your cookies/history. It fixed the issue in my case.
